dll/cpl/appwiz/lang/ro-RO.rc

```diff
@@ -88,8 +89,8 @@ BEGIN
     IDS_DWL_FAILED "Descărcarea pachetului Gecko a eșuat. Asigurați-vă că aveți conexiune la internet pentru a putea descărca. Asistentul va continua fără a instala pachetul Gecko."
     IDS_CANTMAKEINETSHORTCUT "Eșec în crearea scurtăturii la Internet."
     IDS_CANTMAKESHORTCUT "Eșec în crearea scurtăturii."
-    IDS_NO_MEMORY "No memory could be allocated!"
-    IDS_NO_DIRECTORY "No directory given!"
-    IDS_INVALID_PATH "The given path is invalid!"
-    IDS_INVALID_NAME "The shortcut name you entered either contained characters that are invalid for file names or was too long."
+    IDS_NO_MEMORY "Nu s-a putut aloca memorie!"
+    IDS_NO_DIRECTORY "Nu a fost dat niciun director!"
+    IDS_INVALID_PATH "Calea dată este invalidă!"
```

Comment:
Although personally I have no issue with „invalid” in Romanian being used with the same meaning as in English, I must mention that other translators strongly recommend using „nevalid” instead (and leave „invalid” term only for referring to infirm persons).

Reply:
Everything is fixed now.